src/Microsoft.TemplateEngine.Cli/CommandParsing/NewCommandInputCli2.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Edge.Template;
+
+namespace Microsoft.TemplateEngine.Cli.CommandParsing
+{
+    public class NewCommandInputCli2 : INewCommandInput
```

Comment:
We can drop it. I had it because the parser repo calls itself "A second generation of command-line parser for .NET Core command line tools" - so the 2 was for "second", but it's immaterial.